### ISSUE TITLE
Template actions improvements.

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1476]
## Description of issue

Describe what is failing
- Template browser actions should be persistent
- Copy and move are confusing
- 

## Potential fix

- Create and copy template should look like links. Maybe action cards? (Needs design)
- Give context to move buttons and with aria-describedby?
- Allow to move to other services, and remove the Copy link?
- Add controls to each list item, instead of in an action bar. 

## Complete context from OCADU audit:
On the “Browse Templates” page, the “Create Template” and “Copy Template” actions function as links to other pages. However, they are buttons which navigate to those pages when activated. It would be better to have them appear and function as links. In addition, the button (Move template) and link (Copy template), both give the impression that they are actions that can be performed when a template is selected (due to both looking as buttons visually and having button roles). However, while the Move template button behaves in this way, the Copy template button does not, and instead redirects to another page. Consider differentiating between these two actions through presentation and roles, and potentially the name of the controls. 
The Browse Template page has bulk actions which appear when one or more items are selected, replacing the default “Create template”, “Copy template’, and “New folder” actions: “Move” and “Add to New Folder”. When no items are selected, these actions don’t appear. It would be an improvement to show the default actions at all times, even when a selection has been made; if there’s a selection it’s not possible to create a new template without first clearing the selection.
It’s also not possible to directly copy a template from the Browse view; one must navigate to a separate copy templates page, search for the template, and then copy it. It might be beneficial to add a “Copy template” link to each template listed in the Browse view (with visually hidden text that adds the name of the template to the link’s text to ensure that the links are unique, e.g. “Copy template <span class=”sr-only”>Template Name</span>). If this change is made, the default “Copy template” action should be renamed to differentiate it from the individual template actions (for example, “copy template from another service”.
It also may be worth considering adding a “Move template” / “Move folder” actions to each item in the Browse view using the same technique.
